polaris/polaris/management/commands/poll_pending_deposits.py

```diff
@@ -40,13 +41,16 @@ def execute_deposit(transaction: Transaction) -> bool:
             f"Unexpected transaction status: {transaction.status}, expecting "
             f"{' or '.join(valid_statuses)}."
         )
-    if transaction.status != Transaction.STATUS.pending_anchor:
+    if (
+        transaction.status != Transaction.STATUS.pending_anchor
+        and transaction.status != Transaction.STATUS.pending_anchor_claimable_start
```

Comment:
This is an indication that `pending_anchor_claimable_start` is not needed, `pending_anchor` suffices